data/mods/Modular_Turrets/furniture.json

```diff
@@ -11,7 +11,7 @@
     "required_str": 1,
     "flags": [ "TRANSPARENT", "FIRE_CONTAINER", "EASY_DECONSTRUCT" ],
     "deployed_item": "garden_gnome",
-    "examine_action": "deployed_furniture",
+    "examine_action": "fireplace",
```

Comment:
Garden gnome as a place to contain fires? Its `FIRE_CONTAINER` flag looks like a bug to me. It doesn't even have a `max_volume` field to hold flammable material.